### PR TITLE
add BM25s english retrieval results

### PR DESCRIPTION
**Pull request by isaac-chung**
Along with https://github.com/embeddings-benchmark/mteb/pull/1082

### CODE REVIEW

File: results/bm25s/0_1_10/model_meta.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name": "bm25s", "revision": "0_1_10", "release_date": "2024-07-10", "languages": ["eng_Latn"], "n_parameters": null, "memory_usage": null, "max_tokens": null, "embed_dim": null, "license": null, "open_source": true, "similarity_fn_name": null, "framework": [], "loader": "BM25Search"}
```
Comment (isaac-chung) [inline results/bm25s/0_1_10/model_meta.json]: Keep this file or no?

Comment (Muennighoff) [reply results/bm25s/0_1_10/model_meta.json]: I think we can remove it? cc @orionw @KennethEnevoldsen

**Review by Muennighoff — APPROVED** [review]

**Comment by isaac-chung** [comment]
Feel free to merge - I don't think I have permission.

**Comment by Muennighoff** [comment]
Can you still add `CQADupstackRetrieval` by merging the CQADupstack files? (should actually work automatically now but maybe you didn't have these changes in yet: https://github.com/embeddings-benchmark/mteb/pull/1079)

**Comment by isaac-chung** [comment]
Yeah, just merged master and created a model card file. Here's the updated model card with `CQADupstackRetrieval` on line 2335. Should we include this file in the PR?
[model_card.md](https://github.com/user-attachments/files/16238566/model_card.md)

**Comment by Muennighoff** [comment]
Oh, are we not creating a `CQADupstackRetrieval.json` file anymore? We need this to add it to the LB the way it works right now. 🤔 cc'ing @KennethEnevoldsen

**Comment by isaac-chung** [comment]
It's not done automatically when I called create_meta. And it seems like the [old merge script](https://github.com/embeddings-benchmark/mteb/blob/bf02080f6062bc28bff597663ed1759a63233535/scripts/merge_cqadupstack.py) does not work as intended. A new file is created but only saved the results from the last CQADupstackXXX task.

**Comment by isaac-chung** [comment]
[merge.py.zip](https://github.com/user-attachments/files/16246185/merge.py.zip)
Here's the script I modified to work with the current MTEB results format. The `CQADupstackRetrieval.json` is pushed.

When creating the model card, this error is shown:
```
(base) ubuntu@152-67-255-185:~/isaac/work/mteb$ mteb create_meta --results_folder /home/ubuntu/isaac/work/results/results/bm25s/0_1_10 --overwrite
WARNING:mteb.cli:Output path already exists, overwriting.
Traceback (most recent call last):
  File "/home/ubuntu/miniconda3/bin/mteb", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/ubuntu/isaac/work/mteb/mteb/cli.py", line 381, in main
    args.func(args)
  File "/home/ubuntu/isaac/work/mteb/mteb/cli.py", line 270, in create_meta
    task = mteb.get_task(task_result.task_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ubuntu/isaac/work/mteb/mteb/overview.py", line 240, in get_task
    raise KeyError(suggestion)
KeyError: "KeyError: 'CQADupstackRetrieval' not found. Did you mean: CQADupstackTexRetrieval?"
```

**Comment by isaac-chung** [comment]
Based on [this comment](https://github.com/embeddings-benchmark/mteb/pull/1079#issuecomment-2227047324), I'd have thought that aggregating the scores in `create_meta` would be fine. Is that not true anymore?

**Comment by Muennighoff** [comment]
> Based on [this comment](https://github.com/embeddings-benchmark/mteb/pull/1079#issuecomment-2227047324), I'd have thought that aggregating the scores in `create_meta` would be fine. Is that not true anymore?

Sorry that comment only referred to submissions via metadata. Submissions via result files i.e. this repository still need the `CQADupstackRetrieval.json`. Also now that I think about it, we should still allow creating a `CQADupstackRetrieval.json` even when not doing it via metadata in my opinion as people may want to use the results in other ways.

**Comment by isaac-chung** [comment]
Seeing that the`CQADupstackRetrieval.json` is now part of the PR , should we merge this and move the conversation to a separate issue?

**Comment by Muennighoff** [comment]
Yes let's merge! I'm not sure why it says `The base branch does not allow updates` cc @orionw? I will bypass protections, maybe this is a mistake.

Screenshot 2024-07-17 at 2 59 14 PM

**Comment by KennethEnevoldsen** [comment]
> Oh, are we not creating a CQADupstackRetrieval.json file anymore? We need this to add it to the LB the way it works right now. 🤔 cc'ing @KennethEnevoldsen

We no not create it no. It is only created by the create_meta cli.

**Comment by Muennighoff** [comment]
> > Oh, are we not creating a CQADupstackRetrieval.json file anymore? We need this to add it to the LB the way it works right now. 🤔 cc'ing @KennethEnevoldsen
> 
> We no not create it no. It is only created by the create_meta cli.

I see, I think we should automatically create it or at least re-add the script. Unfortunately, it is still needed for any results submitted via PR. What do you think?

**Comment by KennethEnevoldsen** [comment]
> I see, I think we should automatically create it or at least re-add the script. Unfortunately, it is still needed for any results submitted via PR. What do you think?

Re-adding the script seems like a reasonable idea to start of with.

**Pull request merged**